src/app.ts

```diff
@@ -27,7 +27,7 @@ io.on('connection', (sock) => {
     addNewUser(sock.id, sock);
     sock.on('disconnect', async () => {
         logger.info('user disconnected');
-        const newQuantitiesList = await removeUserFromAllItems(sock.id);
+        const newQuantitiesList = await cleanUserCart(sock.id);
```

Comment:
move soket logic to another file and call it from the `www` file - `app.ts` its a module to configure http not ws

Reply:
done